app/components/application_component.rb

```diff
@@ -17,4 +17,27 @@ def merge_classes(*)
   def merge_actions(*)
```

Comment:
I like that we didn't plop `#merge_classes` and `#merge_actions` into a `MergeSupport` class so I thought it worth exploring parity here.